apis/eseries/eseries.go

```diff
@@ -111,7 +118,9 @@ func (d Driver) SendMsg(data []byte, sendType string, msgType string) (*http.Res
 	req.SetBasicAuth(d.config.Username, d.config.Password)
 
 	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: d.config.WebProxyInsecure,
```

Comment:
First we really appreciate you taking the time to contribute your time and code to this project. Very cool to see happening! 

On this line when I tested the code with WebProxyInsecure defaulted to a false value it would force only accepted certificates, and not allow self-signed certificates which cause a panic on my test server:

panic: Post https://localhost:8443/devmgr/v2/storage-systems/: x509: certificate signed by unknown authority

We like the idea of having HTTPS by default, but also think it would be a good idea to make InsecureSkipVerify a configurable setting with it defaulted to a true value (allow self-signed certificates). This would give the user the option to disable self-signed certificates enabling a more secure HTTPS connection.